boot/modeenv.go

```diff
@@ -44,6 +44,10 @@ type Modeenv struct {
 
 	// read is set to true when a modenv was read successfully
 	read bool
+
+	// origindir is set to the location where the modeenv was read from, and
+	// where it will be written back to
+	origindir string
```

Comment:
`originRootDir` since it's the origin root dir, rather than the exact directory `modeenv` is located in?